src/publishers/github.js

```diff
@@ -31,7 +31,8 @@ export default async (artifacts, packageJSON, forgeConfig, authToken, tag) => {
           repo: forgeConfig.github_repository.name,
           tag_name: tag || `v${packageJSON.version}`,
           name: tag || `v${packageJSON.version}`,
-          draft: true,
+          draft: forgeConfig.github_repository.draft !== false,
```

Comment:
Trying to come up with a generic way to pass options in through CLI.  Because we can have multiple publish targets and they might not all support draft or prereleases